pairtools/pairtools_stats.py

```diff
@@ -68,14 +97,25 @@ def stats_py(input_path, output, merge, **kwargs):
     header, body_stream = _headerops.get_header(instream)
     cols = _headerops.extract_column_names(header)
 
+    if (
+        analyse_bytile_dups or output_bytile_dups_stats
+    ) and "parent_readID" not in cols:
+        warnings.warn(
```

Comment:
In pre 0.4.0 we have a new logger! If you decide to merge into pre 0.4.0, then it might be also upgraded.

Reply:
OK, after merging them together...